app/services/data_migrations/backfill_selected_boolean.rb

```diff
@@ -0,0 +1,12 @@
+module DataMigrations
+  class BackfillSelectedBoolean
+    TIMESTAMP = 20210517161321
+    MANUAL_RUN = false
+
+    def change
+      ApplicationReference.feedback_provided.find_in_batches do |batch|
+        batch.each { |reference| reference.update!(selected: true) }
```

Comment:
```suggestion
      ApplicationReference.feedback_provided.in_batches do |batch|
        batch.update_all(selected: true)
```

My mistake, I suggested the wrong method in an earlier comment.

Reply:
I did have a google around and missed in_batches 🤦